### PR TITLE
updated README.md

### PR DESCRIPTION
**Pull request by dersar00**
I had a problem with timeout error when my queue was opened, this description would help me a lot.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,8 @@ iex> AMQP.Basic.publish chan, "test_exchange", "", "Hello, World!"
 Received: Hello, World!
 ```
 
+To avoid problems with terminate in 60 seconds as you opened the connection, pass `heartbet: 30` to `AMQP.Connection.opne()` like argument.
```
Comment (michaelklishin) [inline README.md]: There is a typo in the `open` function name.

Moreoever, the problem definition is very vague. It sounds like a [proxy effect](http://www.rabbitmq.com/heartbeats.html#tcp-proxies) which is environment-specific.

Please reword this to be a generic heartbeat example and link to the [heartbeat guide](http://www.rabbitmq.com/heartbeats.html#tcp-proxies) on rabbitmq.com.

Comment (ono) [reply README.md]: totally agree! thanks :+1:

**Comment by ono** [comment]
Added heartbeats section in Troubleshooting. https://github.com/pma/amqp#heartbeats
